src/devices/Pcf8563/Pcf8563.cs

```diff
@@ -0,0 +1,98 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.I2c;
+
+namespace Iot.Device.Pcf8563
+{
+    /// <summary>
+    /// Realtime Clock PCF8563
+    /// </summary>
+    public class Pcf8563 : IDisposable
+    {
+        private I2cDevice _i2cDevice = null;
```

Comment:
initialization is not needed since you overwrite this in the constructor

Reply:
@krwq Is that critical? I'm used to giving variables a null value. It's the same code style I've submitted. Should I make a unified revision to this?

Reply:
Not critical, but it is one extra instruction, so I would also remove it 😄